proj.4/build.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
 ./configure --prefix=$PREFIX --without-jni
+
 make
+
+if [[ $(uname) == Linux ]]; then
+    make check
+fi
```

Comment:
@QuLogic 

Here is Proj 4.9.2 failure: https://travis-ci.org/ioos/conda-recipes/builds/85402318

And as soon as travis finishes we will know what happens here with proj 4.9.1